lib/dictionary_to_terms/tree_processing.rb

```diff
@@ -450,6 +450,71 @@ def run_tree_flattening_fixed
       Flare.commit
     end
     
+    def run_recording_import(from, to, dialect_id, filename, task_code = nil, force=false, source_dir=nil)
+      exit if filename.nil?
+      require 'recordings/recordings_utils'
+      log = ActiveSupport::Logger.new('log/recordings_importation.log')
+      start_time = Time.now
+      task_code ||= "dtt-recording-import"
+      from = from.nil? ? 0 : from.to_i
+      to = to.nil? ? -1 : to.to_i
+
+      task = ImportationTask.find_by(task_code: task_code)
+      task = ImportationTask.create!(task_code: task_code) if task.nil?
+      self.spreadsheet = task.spreadsheets.find_by(filename: filename)
+      self.spreadsheet = task.spreadsheets.create!(filename: filename, imported_at: start_time) if self.spreadsheet.nil?
+
+      begin
+        dialect = SubjectsIntegration::Feature.find(dialect_id)
+      rescue ActiveRecord::RecordNotFound => e
+        log.error { "Error processing file #{filename} incorrect Dialect ID: #{dialect_id}" }
+        log.close
+        exit
+      end
+      log.info { "#{start_time} From:#{from || "begining"} TO:#{to || "end" } for Dialect: #{dialect_id}" }
+
+      source_dir ||= Rails.root.join('orig_recordings')
+      CSV.foreach( filename, headers: true, col_sep: "\t" ) do |row|
+        rec_id = row["Recording Id"].to_i
+        if rec_id >= from && (to == -1 || rec_id <= to )
+          log.info { "Processing Record: #{rec_id}" }
+          rec_filename = Dir.glob(File.join(source_dir,"*_*#{rec_id}.mp3")).sort.first
+          if !rec_filename.nil? && !rec_filename.empty?
+            begin
+              term = Feature.find(Recordings::RecordingsImporter.map_old_id(row["Dictionary ID"]))
+            rescue ActiveRecord::RecordNotFound => e
+              log.info { "Error processing file No: #{index} - Feature #{file_pattern[:id]} doesn't exist" }
```

Comment:
Change message: 
`Error processing, feature not found with dictionary id: #{row["Dictionary ID"]}"`